actions/log-telemetry-event/action.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Log Telemetry Event"
+description: "Append a telemetry event to the session file"
```

Comment:
suggestion: what's the "session file"? I'd try to be more explicit in this description

Reply:
It's documented on the readme file